examples/communication/child_to_parent/src/main.rs

```diff
@@ -0,0 +1,86 @@
+use yew::{html, AttrValue, Callback, Component, Context, Html, Properties};
+
+// The `Parent` component is the parent of the `Child` component.
+
+pub struct Parent {
```

Comment:
This comment contains little information. Consider making into a normal doc comment on Parent and rewording it.

```suggestion
/// The `Parent` component holds some state that is updated when its children are clicked
pub struct Parent {
```

Reply:
Done